package.json

```diff
@@ -9,13 +9,14 @@
     "dist"
   ],
   "scripts": {
-    "dev": "vite --open",
+    "prebuild": "node ./scripts/wrapper-pre-build.cjs",
+    "dev": "npm run prebuild && vite --open",
     "build": "tsc --p ./tsconfig-build.json && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
     "build:storybook": "storybook build",
-    "build:cdn": "tsc --p ./tsconfig-build.json && vite --config cdn-vite.config.ts build",
+    "build:cdn": "tsc --p ./tsconfig-build.json && vite --config cdn-vite.config.ts build && node ./scripts/cdn-post-build.cjs",
```

Comment:
this part `node ./scripts/cdn-post-build.cjs` could also be in a postbuild:cdn script

Reply:
Good spot! I'll also add both of these to the README